src/include/metrics/pipeline_metric.h

```diff
@@ -77,8 +78,8 @@ class PipelineMetricRawData : public AbstractRawData {
    * Note: This includes the columns for the input feature, but not the output (resource counters)
    */
   static constexpr std::array<std::string_view, 1> FEATURE_COLUMNS = {
-      "query_id, pipeline_id, exec_mode, num_features, features, est_output_rows, key_sizes, num_keys, "
-      "est_cardinalities, mem_factor, num_loops"};
+      "query_id, pipeline_id, num_features, features, exec_mode, est_output_rows, key_sizes, num_keys, "
```

Comment:
I believe this is no longer consistent with data_info.py 's `RAW_FEATURES_VECTOR_INDEX = 4`

Reply:
That is correct. I have some unpushed local changes aimed at refactoring away all the indexes.